--- ./includes/addressfield_form_default.inc	2012-01-04 08:55:36.447146773 +0100
+++ ../../../external/drupal/modules/addressfield/includes/addressfield_form_default.inc	2012-01-04 15:17:22.953443260 +0100
@@ -14,7 +14,7 @@ class AddressFieldForm_default implement
       '#size' => 30,
       '#default_value' => $address['thoroughfare'],
     );
-/*    $form['premise'] = array(
+    $form['premise'] = array(
       '#title' => t('Residence, building, ...'),
       '#type' => 'textfield',
       '#size' => 30,
@@ -32,7 +32,7 @@ class AddressFieldForm_default implement
       '#size' => 30,
       '#default_value' => $address['dependent_locality'],
     );
-*/    $form['postal_code'] = array(
+    $form['postal_code'] = array(
       '#title' => t('Postal Code'),
       '#type' => 'textfield',
       '#size' => 10,
