diff --git a/terms_of_use.admin.inc b/terms_of_use.admin.inc
index 59ca703..9a35ae2 100644
--- a/terms_of_use.admin.inc
+++ b/terms_of_use.admin.inc
@@ -23,7 +23,7 @@ function terms_of_use_admin_settings() {
     '#title' => t('Title of the post where your Terms of Use are published'),
     '#default_value' => variable_get('terms_of_use_node_title', ''),
     '#description' => t('Node <em>title</em> of the page or story (or blog entry or book page) where your Terms of Use are published.'),
-    '#autocomplete_path' => 'node/autocomplete',
+    '#autocomplete_path' => 'terms_of_use/autocomplete',
   );
   $form['terms_of_use_text']['terms_of_use_pick_node_id'] = array(
     '#type' => 'button',
@@ -144,7 +144,7 @@ function terms_of_use_js($form, &$form_state) {
       '#title' => t('Title of the post where your Terms of Use are published'),
       '#default_value' => variable_get('terms_of_use_node_title', ''),
       '#description' => t('Node <em>title</em> of the page or story (or blog entry or book page) where your Terms of Use are published.'),
-      '#autocomplete_path' => 'node/autocomplete',
+      '#autocomplete_path' => 'terms_of_use/autocomplete',
     );
     unset($form['terms_of_use_text']['terms_of_use_node_id']);
     $form['terms_of_use_text']['terms_of_use_pick_node_id']['#value'] = t('I prefer to specify the node id');
